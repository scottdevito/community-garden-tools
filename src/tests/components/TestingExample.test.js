import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { shallow, mount, render } from 'enzyme';

import TestingExample from '../../components/TestingExample';

// Snapshot test
it('renders correctly', () => {
  const tree = renderer.create(<TestingExample />).toJSON();
  expect(tree).toMatchSnapshot();
});
