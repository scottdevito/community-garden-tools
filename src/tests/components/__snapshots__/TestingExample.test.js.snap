// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-EHOje cLeeWa"
>
  <h1>
    Testing
  </h1>
  <div
    className="sc-ifAKCX eVrQyi"
  >
    <h2>
      Resources:
    </h2>
    <h5>
      Snapshot
    </h5>
    <a
      href="https://jestjs.io/docs/en/snapshot-testing"
    >
      https://jestjs.io/docs/en/snapshot-testing
    </a>
    <a
      href="https://www.youtube.com/watch?v=9liG4PuqKY0"
    >
      https://www.youtube.com/watch?v=9liG4PuqKY0
    </a>
    <h5>
      Unit Tests
    </h5>
    <a
      href="https://github.com/airbnb/enzyme"
    >
      https://github.com/airbnb/enzyme
    </a>
    <a
      href="https://airbnb.io/enzyme/docs/installation/index.html"
    >
      https://airbnb.io/enzyme/docs/installation/index.html
    </a>
  </div>
</div>
`;
